modules/microshift-ingress-controller-config.adoc

```diff
@@ -122,8 +244,7 @@ When configured, this field must contain a valid expression or the {microshift-s
 |`Required` or `Optional` are valid values. Set to `Required` to enable client TLS. The ingress controller only checks client certificates for edge-terminated and re-encrypted TLS routes. The ingress controller cannot check certificates for plain text HTTP or passthrough TLS routes.
 
 |`defaultHTTPVersion`
-|Sets the HTTP version for the ingress controller. Default value is `1` for HTTP 1.1.
-//Q: do we need to configure a load balancer for 2 and 3?
+|Sets the HTTP version for the ingress controller. The default value is `1` for HTTP 1.1. Setting up a load balancer for HTTP 2 and 3 is recommended.
 
 |`forwardedHeaderPolicy`
 |Specifies when and how the ingress controller sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers. The following values are valid:
```

Comment:
I can't directly comment on it here, but on line 252, the second instance of "Append" is not rendering correctly in the preview.